.github/workflows/test_and_release.yml

```diff
@@ -102,6 +102,7 @@ jobs:
           dpns-top-level-identity-id: ${{ env.dpns-top-level-identity-id }}
           dpns-top-level-identity-private-key: ${{ env.dpns-top-level-identity-private-key }}
           platform-test-suite-branch: ${{ steps.platform-branches.outputs.testsuite-branch }}
+          scope: platform
```

Comment:
We didn't have scope for DAPI and I'm not sure we need it now

Reply:
Scope was [defined in `.travis.yml` here](https://github.com/dashevo/dapi/pull/353/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L58) for DAPI. What does `scope` actually do, what are the possible options?

Reply:
Oh, you right. Sorry.

The `scope` is an option of test suite which allows you to set which tests should be run. If the option is omitted - all tests will be run. Different repositories affect different tests so for some of them you just don't need to run all of them to reduce build time.

Actually, DAPI affects all tests so we don't need the scope here.

Reply:
Should I remove the default "platform" scope from [the action here](https://github.com/dashevo/gh-action-run-platform-test-suite/blob/master/action.yml#L19) then? In order to run all tests, should the scope be undefined or set with `scope=`?

Reply:
Yes, we need to remove the default value. Not sure about the empty scope. You need to check this logic in the test suite's `test.sh`

Reply:
OK, it looks like it will run all tests (`functional` and `e2e`, which includes `platform`) if scope is empty.

Reply:
Great! Let’s remove platform from here then :))